src/pages/Environments.js

```diff
@@ -158,6 +160,14 @@ const Environments = ({ namespace }) => {
 
   const loadData = withErrorReporting('Error loading cloud environments', refreshData)
 
+  const pauseComputeAndRefresh = withErrorReporting('Error loading cloud environments', async (computeType, compute) => {
+    const wrappedPauseCompute = withBusyState(setLoading, () => computeType === 'runtime' ?
+      Ajax().Runtimes.runtime(compute.googleProject, compute.runtimeName).stop() :
+      Ajax().Apps.app(compute.googleProject, compute.appName).pause())
+    await wrappedPauseCompute()
+    refreshData()
+  })
```

Comment:
```suggestion
  const pauseComputeAndRefresh = withBusyState(setLoading, async (computeType, compute) => {
    const wrappedPauseCompute = withErrorReporting('Error stopping compute', () => computeType === 'runtime' ?
      Ajax().Runtimes.runtime(compute.googleProject, compute.runtimeName).stop() :
      Ajax().Apps.app(compute.googleProject, compute.appName).pause())
    await wrappedPauseCompute()
    await loadData()
  })
```
but also wrap error more specifically for stop/pause